### PR TITLE
Validate and deploy supplemental prow config

### PR DESCRIPTION
**Pull request by alvaroaleman**
Ref https://issues.redhat.com/browse/DPTP-2072

Shouldn't do anything yet as there is no supplemental config so far.

/hold
To be sure rehearsal passes
/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml [hunk 1]
```diff
@@ -442,6 +442,8 @@ presubmits:
       - args:
         - --config-path
         - ./core-services/prow/02_config/_config.yaml
+        - --supplemental-prow-config-dir
+        - core-services/prow/02_config
```
Comment (hongkailiu) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:446]:
```suggestion
        - ./core-services/prow/02_config
```

To keep the consistency?

Comment (alvaroaleman) [reply ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:446]: I had that initially and that makes checkconfig fail :/

File: core-services/prow/02_config/_plugins.yaml [hunk 2]
```diff
@@ -2851,6 +2851,12 @@ config_updater:
         app.ci:
         - ci
       name: image-mirror-mappings
+    core-services/prow/02_config/**/_prowconfig.yaml:
+      clusters:
+        app.ci:
+        - ci
+      name: config
```
Comment (hongkailiu) [inline core-services/prow/02_config/_plugins.yaml:2858]:
```
$ oc get cm -n ci config
NAME     DATA   AGE
config   1      368d
```

Name conflicting?

Comment (alvaroaleman) [reply core-services/prow/02_config/_plugins.yaml:2858]: That is intentional, it all needs to go into the same configmap

**Review by hongkailiu — APPROVED** [review]

**Pull request merged**